src/AcceptanceTests/Routing/When_subscribing_outside_the_endpoint_oriented_topology.cs

```diff
@@ -0,0 +1,113 @@
+﻿namespace NServiceBus.Azure.Transports.WindowsAzureServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Support;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using Features;
+    using NServiceBus.AcceptanceTests.ScenarioDescriptors;
+    using NUnit.Framework;
+
+    public class When_subscribing_outside_the_endpoint_oriented_topology : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_receive_event()
+        {
+            var topology = EnvironmentHelper.GetEnvironmentVariable("AzureServiceBusTransport.Topology");
+            if (topology == "ForwardingTopology")
+            {
+                Assert.Inconclusive("The test is designed for EndpointOrientedTopology only.");
+            }
+
+            var runSettings = new RunSettings();
+            runSettings.TestExecutionTimeout = TimeSpan.FromMinutes(1);
+
+            var config = new AzureServiceBusTransportConfigContext();
+            config.Callback = (endpointName, extensions) =>
+            {
+                var publisherConnectionString = EnvironmentHelper.GetEnvironmentVariable("AzureServiceBusTransport.ConnectionString");
+                var subscriberConnectionString = EnvironmentHelper.GetEnvironmentVariable("AzureServiceBus.ConnectionString.Fallback");
+
+                if (endpointName == "SubscribingOutsideTheEndpointOrientedTopology.Subscriber")
+                {
+
+                    extensions.NamespacePartitioning().AddNamespace("subscriberNamespace", subscriberConnectionString);
+                    extensions.NamespaceRouting()
+                                    .AddNamespace("publisherNamespace", publisherConnectionString)
+                                    .RegisteredEndpoints.Add(AcceptanceTesting.Customization.Conventions.EndpointNamingConvention(typeof(Publisher)));
```

Comment:
Would suggest to take the namespaceInfo returned by `extensions.NamespaceRouting().AddNamespace()` and assign to a variable used along with `.RegisteredEndpoints.Add()`. Fluent API while correct, harder to digest.